### ISSUE TITLE
Rebind popup commands?

### ISSUE THREAD

**Issue by lucasdf** [iamcco/coc-actions#11]
In `coc-nvim` we can use `pumvisible() ` to detect whether the popup menu is active and change the mappings used for completion control. 

For instance, I use `tab` instead of `<CR>` or `<c-y>` to confirm selection using something like this:
```
inoremap <silent><expr> <TAB> 
  \ pumvisible() ? "\<C-y>" : 
```

The same doesn't seem to work here. It seems the completion menu is not a vim's completion menu? Is there a way to remap the keys and do the same with `coc-actions`?

**Comment by iamcco** [comment]
Yes, it is not a completion menu, it is floating window. Maybe add some options to support this.
